lib/src/realm_class.dart

```diff
@@ -28,10 +28,18 @@ import 'list.dart';
 
 export 'list.dart' hide RealmListInternal;
 export 'results.dart' hide RealmResultsInternal;
+
+export 'realm_object.dart' show RealmException, RealmObject;
 export 'realm_object.dart'
     hide RealmObjectInternal, RealmAccessor, RealmValuesAccessor, RealmMetadata, RealmCoreAccessor, RealmClassMetadata, RealmPropertyMetadata;
+
+export "configuration.dart" show Configuration, RealmSchema, SchemaObject;
 export "configuration.dart" hide ConfigurationInternal;
-export 'package:realm_common/realm_common.dart' show RealmModel, PrimaryKey, Ignored, MapTo, Indexed, RealmPropertyType, RealmCollectionType;
+
+export 'package:realm_common/realm_common.dart'
+    show Ignored, Indexed, MapTo, RealmError, RealmModel, RealmUnsupportedSetError, RealmCollectionType, RealmPropertyType;
```

Comment:
I don't see PrimaryKey.

Reply:
good catch

Reply:
Hmm.. why does it work then 🤔

Reply:
Well, that kind of prove your point doesn't it 😄 .. I have removed the hide, and only use show now